src/solvers/dg_tree/dg_2d.jl

```diff
@@ -901,7 +901,8 @@ function calc_mortar_flux!(surface_flux_values,
     calc_fstar!(fstar_upper, equations, dg, u_upper, mortar, orientation)
     calc_fstar!(fstar_lower, equations, dg, u_lower, mortar, orientation)
 
-    mortar_fluxes_to_elements!(surface_flux_values, equations, mortar_l2, dg, cache,
+    mortar_fluxes_to_elements!(surface_flux_values, mortar_l2,
+                               mesh, equations, dg, cache,
```

Comment:
Can you please fix the argument order as discussed (I also found the relevant section in the docs: https://trixi-framework.github.io/Trixi.jl/stable/styleguide/)